website/sidebars.json

```diff
@@ -647,6 +647,7 @@
 					"label": "App Design",
 					"items": [
 						"react-native/styles",
+						"react-native/custom-icon-font",
```

Comment:
Please ensure the filename, id, and sidebar reference are matching. 

Broken link: https://pr-858.d2ulx3keu6fdjg.amplifyapp.com/learn/react-native/custom-icon-fonts/